bin.src/makeGainImages.py

```diff
@@ -47,21 +47,21 @@ def main(just_wfs=False, detector_list=None):
     for filt_name in 'ugrizy':
         for ccd in ccd_list:
             name = ccd.getName()
+            print(f'Preparing gain images in filter {filt_name} for {name}')
```

Comment:
I thought the `black` will use the double quote?

```
print(f"Preparing gain images in filter {filt_name} for {name}")
```

Reply:
Yes, it doesn't seem that black had been run on this file for a while; when I run it it changes that also in all `.set("name")` methods